add-convo-record-for-every-convo.py

```diff
@@ -0,0 +1,21 @@
+from sqlite3 import DatabaseError
+
+from sqlalchemy import create_engine, insert
+from sqlalchemy.orm import sessionmaker
+
+from config import Config
+from secure_message.repository.database import Conversation, SecureMessage
+
+engine = create_engine(Config.SQLALCHEMY_DATABASE_URI)
+Session = sessionmaker(bind=engine)
+db = Session()
+
+try:
+    subquery = db.query(Conversation.id)
+    distinct_ids = db.query(SecureMessage.thread_id).filter(~SecureMessage.thread_id.in_(subquery)).distinct()
+    for thread_id in distinct_ids:
+        db.execute(insert(Conversation).values({"id": thread_id, "is_closed": "False"}))
+    db.commit()
+except DatabaseError:
+    db.rollback()
+    print(DatabaseError)
```

Comment:
We should exit the script with an error (see https://stackoverflow.com/questions/22633544/how-to-throw-error-and-exit-in-python). This means when it's run as a CloudFoundry task if it errors it will be visible in the task list as a failure (https://docs.cloudfoundry.org/devguide/using-tasks.html#list-tasks)